Makefile

```diff
@@ -6,6 +6,18 @@ build:
 	go build -o image-builder-migrate-db-tern ./cmd/image-builder-migrate-db-tern/
 	go test -c -tags=integration -o image-builder-db-test ./cmd/image-builder-db-test/
 
+.PHONY: test
+test:
+	go test ./...
+
+.PHONY: generate
+generate:
+	go generate ./...
+
+.PHONY: fmt
+fmt:
+	go fmt ./...
+
```

Comment:
Two of these are in the `tools/prepare-source.sh` tho I really was not searching in there.

Reply:
If we want these targets in the `Makefile`, they should be running in a similar way as `tools/prepare-source.sh` does: Enforcing a specific version of Go. Otherwise, they might not actually format the code according to the same standards that our CI enforces.

I'm actually against adding these, `tools/prepare-source.sh` already does everything we need. One potential improvement might to be to add a `prep` target to the `Makefile` calling `tools/prepare-source.sh`, but I don't see that much value in it.

Reply:
I agree here, either we could have a `make prepare-source` target, but we need to enforce the go version each time.